Maple2Storage/Types/Metadata/MapEntityMetadata.cs

```diff
@@ -252,46 +281,72 @@ public class MapPortal
         [XmlElement(Order = 2)]
         public readonly string Name;
         [XmlElement(Order = 3)]
-        public readonly MapPortalFlag Flags;
+        public readonly bool Enable;
         [XmlElement(Order = 4)]
-        public readonly int Target;
+        public readonly bool IsVisible;
         [XmlElement(Order = 5)]
-        public readonly CoordS Coord;
+        public readonly bool MinimapVisible;
         [XmlElement(Order = 6)]
-        public readonly CoordS Rotation;
+        public readonly int Target;
         [XmlElement(Order = 7)]
-        public readonly int TargetPortalId;
+        public readonly CoordS Coord;
         [XmlElement(Order = 8)]
+        public readonly CoordS Rotation;
+        [XmlElement(Order = 9)]
+        public readonly int TargetPortalId;
+        [XmlElement(Order = 10)]
         public readonly byte PortalType;
+        [XmlElement(Order = 11)]
+        public readonly int TriggerId;
 
         // Required for deserialization
         public MapPortal() { }
 
-        public MapPortal(int id, string name, MapPortalFlag flags, int target, CoordS coord, CoordS rotation, int targetPortalId, byte portalType)
+        public MapPortal(int id, string name, bool enable, bool isVisible, bool minimapVisible, int target, CoordS coord, CoordS rotation, int targetPortalId, byte portalType)
```

Comment:
just add `int triggerId` with an default value to the other constructor

Reply:
done